### PR TITLE
[DOCU-685] mTLS plugin add datatypes, formatting, editing

### PR DESCRIPTION
**Pull request by tlblessing**
https://konghq.atlassian.net/browse/DOCU-685 as part of epic https://konghq.atlassian.net/browse/DOCU-301. The doc jira board shuffling/renaming has deleted links to child tickets.

Schema link:

https://github.com/Kong/kong-plugin-mtls-auth/blob/master/kong/plugins/mtls-auth/schema.lua

Direct review link:

https://deploy-preview-2751--kongdocs.netlify.app/hub/kong-inc/mtls-auth/

### CODE REVIEW

File: app/_hub/kong-inc/mtls-auth/index.md [hunk 1]
```diff
@@ -29,44 +29,78 @@ params:
   name: mtls-auth
   service_id: true
   route_id: true
+  protocols: ["http", "https", "grpc", "grpcs", "tcp", "tls", "udp"]
```
Comment (tlblessing) [inline app/_hub/kong-inc/mtls-auth/index.md:32]: @murillopaula need to verify protocols, not sure, schema not definitive

File: app/_hub/kong-inc/mtls-auth/index.md [hunk 2]
```diff
@@ -29,44 +29,78 @@ params:
...
       description: |
-        HTTP timeout threshold, in milliseconds, when communicating with the OCSP server or downloading CRL.
+        HTTP timeout threshold in milliseconds when communicating with the OCSP server or downloading CRL.
     - name: cert_cache_ttl
       default: "60000"
+      datatype: number
       description: |
-        The length of time, in milliseconds, between refreshes of the revocation check status cache.
+        The length of time in milliseconds between refreshes of the revocation check status cache.
+    - name: cache_ttl
```
Comment (tlblessing) [inline app/_hub/kong-inc/mtls-auth/index.md:97]: @murillopaula this cache_ttl has been missing for a few years. Added and used LDAP's description. Please verify.

File: app/_hub/kong-inc/mtls-auth/index.md (hunk 1 above)
Comment (murillopaula) [inline app/_hub/kong-inc/mtls-auth/index.md:32]: `typedefs.http_protocols` are defined here: https://github.com/Kong/kong/blob/master/kong/constants.lua#L53-L61

```suggestion
  protocols: ["http", "https", "grpc", "grpcs"]
```

File: app/_hub/kong-inc/mtls-auth/index.md (hunk 2 above)
Comment (murillopaula) [reply app/_hub/kong-inc/mtls-auth/index.md:97]: Very nice catch here. Verified.

**Review by murillopaula — APPROVED** [review]

**Pull request merged**